sdk/include/ucrt/corecrt.h

```diff
@@ -280,7 +280,18 @@ extern "C++"
 #define _CRT_SIZE_MAX ((size_t)-1)
 
 #define __FILEW__     _CRT_WIDE(__FILE__)
+
+#ifdef _MSC_VER
 #define __FUNCTIONW__ _CRT_WIDE(__FUNCTION__)
+#else // _MSC_VER
```

Comment:
What kind of error does GCC emit? Isn't `__FUNCTION__` available there?

Reply:
__FUNCTION__ is not a preprocessor token in GCC but a string literal, so it cannot be converted to wide with the preprocessor.